index.css

```diff
@@ -1,53 +1,63 @@
+*{
+    margin: 0;
+    padding: 0;
+}
 body{
     background-color: aliceblue;
    display: flex;
    justify-content: center;
 }
 .main-container{
-    background-color: rgb(236, 230, 230);
-    height: 100%;
-    width: 20%;
```

Comment:
Kindly avoid using fixed percentages for width and height in order to put into account all screen sizes. For instance, this would be fine for desktops but 20% on a mobile screen gives it a distorted look